test/junit/scala/collection/IteratorTest.scala

```diff
@@ -1,6 +1,6 @@
 package scala.collection
 
-import org.junit.Assert._
+import org.junit.Assert.{ assertThrows => _, _ }
```

Comment:
I see you also like spacey braces.

Alternatively, add the specific import from testkit.

There's also the thing where junit is at 5 now. I wonder what that bump entails?

Reply:
spacey braces are ace!

> Alternatively, add the specific import from testkit

yeah, I went with the mechanically applicable resolution here rather than go artisanal

> There's also the thing where junit is at 5 now. I wonder what that bump entails?

personally I'm content if we stay on 4